src/Symfony/Component/Validator/Constraints/LengthValidator.php

```diff
@@ -65,7 +65,12 @@ public function validate(mixed $value, Constraint $constraint)
             return;
         }
 
-        $length = mb_strlen($stringValue, $constraint->charset);
+        $length = match ($constraint->countUnit) {
+            Length::COUNT_BYTES => \strlen($stringValue),
+            Length::COUNT_CODEPOINTS => \mb_strlen($stringValue, $constraint->charset),
+            Length::COUNT_GRAPHEMES => \grapheme_strlen($stringValue),
+            default => throw new \RuntimeException(sprintf('That countUnit is not implemented in "%s", "%s" should not have allowed that value at all.', __CLASS__, Length::class)),
```

Comment:
I suggest removing the `default` entirely, as PHP always validates that `match` is exhaustive and we already validate the value in the constraint (so this code is unreachable).